scripts/secure-boot/generate-images/intermediate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# go into a separate directory to make cleanup easier
+mkdir -p something
+cd something
+
+# make the new image
+dd if=/dev/zero of=image.img bs=1M count=8
+mkfs.fat -F 12 image.img
+
+# generate the first key pair
+openssl req -new -x509 -newkey rsa:2048 -subj "/CN=PK/" -keyout intermediate-db.key -out intermediate-db.crt -days 3650 -nodes -sha256
+
+# generate the second key pair
+openssl req -new -x509 -newkey rsa:2048 -subj "/CN=PK/" -keyout PK.key -out PK.crt -days 3650 -nodes -sha256
+
+# make the CSR, its necessary to input something (country=PL, company=3mdeb)
+openssl req -new -key intermediate-db.key -out intermediate-db.csr -subj "/C=PL/O=3mdeb"
+
+# sign the CSR with the second key pair
+# its necessary to `touch intermediate-db.ext`, otherwise it says its not there
+touch intermediate-db.ext
+openssl x509 -req -in intermediate-db.csr -CA PK.crt -CAkey PK.key -out intermediate-db.crt -CAcreateserial -extfile intermediate-db.ext
+
+# convert the cert which we will enroll to the right format
+openssl x509 -in intermediate-db.crt -outform der -out intermediate-db.der
+
+# sign hello.efi
+cp ../scripts/secure-boot/generate-images/hello.efi .
+sbsign --key intermediate-db.key --cert intermediate-db.crt --output signed-hello.efi hello.efi
+
+# copy the necessary things to the image
+mkdir -p mnt/
```

Comment:
@WiktorG351 can we mount it under another folder created with mktemp ?

Reply:
@TomaszAIR since the parent folder is now being made with `mktemp -d`, then it should be fine to keep it as it (?)